practice-app/templates/coronavirus.html

```diff
@@ -105,19 +105,13 @@
                 </div>
                 <nav class="navbar">
                   <ul class="navbar_menu">
-                    <li><a href="#">home</a></li>
+                    <li><a href="/">home</a></li>
                   </ul>
                   <ul class="navbar_user">
-                    <li>
-                      <a href="#"
-                        ><i class="fa fa-search" aria-hidden="true"></i
-                      ></a>
-                    </li>
-                    <li>
-                      <a href="#"
-                        ><i class="fa fa-user" aria-hidden="true"></i
-                      ></a>
-                    </li>
+                     <li><a href="coronavirusCountryLive">Search Country</a></li>
+                  </ul>
+		  <ul class="navbar_user">
+                     <li><a href="worldStats">Worldwide Overview</a></li>
```

Comment:
Same applies here.

Reply:
i checked  usage of {{ url_for() }} function, but my code seems fine too. It uses relative path linking, thank you for your review.